argo-cd-apps/base/member/infra-deployments/backup/backup.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: backup
+spec:
+  generators:
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
+              values:
+                sourceRoot: components/backup
+                environment: staging
+                clusterDir: base
+          - list:
+              elements:
+                - nameNormalized: stone-stg-m01
+                  values.clusterDir: stone-stg-m01
+                - nameNormalized: stone-stg-rh01
+                  values.clusterDir: stone-stg-rh01
```

Instead of hardcoding the cluster names here, let's change this block this add apply the comment in posted in sibling kustomization.yaml file:

```suggestion
spec:
  generators:
    - merge:
        mergeKeys:
          - nameNormalized
        generators:
          - clusters:
              values:
                sourceRoot: components/backup
                environment: staging
                clusterDir: ""
          - list:
              elements: []
```